gbif/ingestion/ingest-gbif-beam/src/main/resources/elasticsearch/es-validator-schema.json

```diff
@@ -1,38 +1,13 @@
 {
     "dynamic_templates": [
       {
-        "verbatim_string": {
+        "verbatim": {
           "path_match": "verbatim.*",
-          "match_mapping_type": "string",
-          "mapping": {  "type": "keyword" }
+          "mapping": {
+            "type": "keyword",
+            "normalizer": "lowercase_normalizer",
+            "copy_to": "all"
```

Comment:
do they need to be copied to all?

Reply:
ive reverted a lot of these changes to be more inline with dev